lib/money-rails/helpers/action_view_extension.rb

```diff
@@ -16,10 +16,12 @@ def humanized_money(value, options={})
         :symbol => false
       }.merge(options)
       options.delete(:symbol) if options[:disambiguate]
-
+      
       if value.is_a?(Money)
+        value = value.to_d.round.to_money if options[:no_cents] == true && value.present?
         value.format(options)
       elsif value.respond_to?(:to_money)
+        value = value.to_d.round if options[:no_cents] == true && value.present?
```

Comment:
Similarly here, I think  `value.respond_to?(:to_d)` would be more rigorous and gets to the heart of what this line cares about than `value.present?`.